lib/plugins/dev-server.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const pathUtil = require('../config/path-util');
+const AssetOutputDisplayPlugin = require('../friendly-errors/asset-output-display-plugin');
+const FriendlyErrorsPlugin = require('./friendly-errors');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @param {Array} paths to clean
+ * @param {Object} cleanUpOptions
+ * @return {Array} of plugins to add to webpack
+ */
+module.exports = {
+    getPlugins(webpackConfig) {
+        const outputPath = pathUtil.getRelativeOutputPath(webpackConfig);
+        const friendlyErrorsPlugin = FriendlyErrorsPlugin.getPlugins();
+        return [new AssetOutputDisplayPlugin(outputPath, friendlyErrorsPlugin[0])];
```

Comment:
This is clever, but a problem :). Because `FriendlyErrorsPlugin.getPlugins()` actually instantiates a `new FriendlyErrorsWebpackPlugin` internally, when `FriendlyErrorsPlugin.getPlugins()` is called by `config-generator`, it will receive one instance. When it's called here, it will receive a different instance. I "got away: with doing stuff like this with the loaders, because those return static config objects, not new instances of objects. In practice, this might not be a problem, but we can do better :).

The only idea I can think of right now, is to change `plugins` in `config-generator` to a hash - i.e. create keys for each plugin - e.g.

```js
// old
plugins.push(new FriendlyErrorsPlugin());

// now
plugins.push('friendly_errors_plugin', new FriendlyErrorsPlugin());
```

then we could use that internal key to find other plugins. At the bottom of `buildPluginsConfig()` in `config-generator`, we would still return an actual array (not a hash/object). I don't love this magic, invented `friendly_errors_plugin` machine name... but it would work, and would still be an internal implementation detail, at least for now (i.e. not exposed to users in any way).